ImperatorToCK3/Source/CK3/Titles/Title.cpp

```diff
@@ -268,6 +268,15 @@ void CK3::Title::setDeFactoLiege(const std::shared_ptr<Title>& liegeTitle) {
 }
 
 
+const std::optional<int>&  CK3::Title::getOwnOrInheritedDevelopmentLevel() const {
+	if (history.developmentLevel) // if development level is already set, just return it
+		return history.developmentLevel;
+	if (deJureLiege) //if de jure liege exists, return their level
+		return deJureLiege->getOwnOrInheritedDevelopmentLevel();
+	return history.developmentLevel; // if there's no other choice, return own nullopt level
```

Comment:
I think it'd be a little more clear if you explicitly return std::nullopt here instead of relying on the stored nullopt.

Reply:
That way I would need to return by value, no?

Reply:
Maybe? But even if so, clarity is better than efficiency. Especially since a std::optional<int> is going to be pretty lightweight regardless. No need to worry about a copy, really.